Xamarin.Forms.Xaml.UnitTests/Issues/Gh4319.xaml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should there also be a unit test for this with `OnIdiom`? Or is that covered elsewhere?